environments/icds-cas/inventory/postgresql.csv

```diff
@@ -1,113 +1,112 @@
+pgshard24,100.71.184.175,postgresql,shard_dbs,,MUMGCCWCDPRDPS24,pgshard24standby,,"[""standby0"",""spare""]",,,
+pgshard25,100.71.184.176,postgresql,shard_dbs,,MUMGCCWCDPRDPS25,pgshard25standby,,"[""standby0"",""spare""]",,,
+pgshard26,100.71.184.177,postgresql,shard_dbs,,MUMGCCWCDPRDPS26,pgshard26standby,,"[""standby0"",""spare""]",,,
+pgshard27,100.71.184.178,postgresql,shard_dbs,,MUMGCCWCDPRDPS27,pgshard27standby,,"[""standby0"",""spare""]",,,
+pgshard28,100.71.184.179,postgresql,shard_dbs,,MUMGCCWCDPRDPS28,pgshard28standby,,"[""standby0"",""spare""]",,,
+pgshard29,100.71.184.180,postgresql,shard_dbs,,MUMGCCWCDPRDPS29,pgshard29standby,,"[""standby0"",""spare""]",,,
+pgshard10standby,100.71.184.161,pg_standby,shard_dbs,,MUMGCCWCDPRDSS10,,pgshard10,,standby0,,
+pgshard11standby,100.71.184.162,pg_standby,shard_dbs,,MUMGCCWCDPRDSS11,,pgshard11,,standby0,,
+pgshard12standby,100.71.184.163,pg_standby,shard_dbs,,MUMGCCWCDPRDSS12,,pgshard12,,standby0,,
+pgshard13standby,100.71.184.164,pg_standby,shard_dbs,,MUMGCCWCDPRDSS13,,pgshard13,,standby0,,
+pgshard14standby,100.71.184.165,pg_standby,shard_dbs,,MUMGCCWCDPRDSS14,,pgshard14,,standby0,,
+pgshard15standby,100.71.184.166,pg_standby,shard_dbs,,MUMGCCWCDPRDSS15,,pgshard15,,standby0,,
+pgshard16standby,100.71.184.167,pg_standby,shard_dbs,,MUMGCCWCDPRDSS16,,pgshard16,,standby0,,
+pgshard17standby,100.71.184.168,pg_standby,shard_dbs,,MUMGCCWCDPRDSS17,,pgshard17,,standby0,,
+pgshard18standby,100.71.184.169,pg_standby,shard_dbs,,MUMGCCWCDPRDSS18,,pgshard18,,standby0,,
+pgshard19standby,100.71.184.170,pg_standby,shard_dbs,,MUMGCCWCDPRDSS19,,pgshard19,,standby0,,
+pgshard20standby,100.71.184.181,pg_standby,shard_dbs,,MUMGCCWCDPRDSS20,,pgshard20,,standby0,,
+pgshard21standby,100.71.184.182,pg_standby,shard_dbs,,MUMGCCWCDPRDSS21,,pgshard21,,standby0,,
+pgshard22standby,100.71.184.183,pg_standby,shard_dbs,,MUMGCCWCDPRDSS22,,pgshard22,,standby0,,
+pgshard23standby,100.71.184.184,pg_standby,shard_dbs,,MUMGCCWCDPRDSS23,,pgshard23,,standby0,,
+pgshard24standby,100.71.184.185,pg_standby,shard_dbs,,MUMGCCWCDPRDSS24,,pgshard24,,standby0,,
+pgshard25standby,100.71.184.186,pg_standby,shard_dbs,,MUMGCCWCDPRDSS25,,pgshard25,,standby0,,
+pgshard26standby,100.71.184.187,pg_standby,shard_dbs,,MUMGCCWCDPRDSS26,,pgshard26,,standby0,,
+pgshard27standby,100.71.184.188,pg_standby,shard_dbs,,MUMGCCWCDPRDSS27,,pgshard27,,standby0,,
+pgshard28standby,100.71.184.189,pg_standby,shard_dbs,,MUMGCCWCDPRDSS28,,pgshard28,,standby0,,
+pgshard29standby,100.71.184.190,pg_standby,shard_dbs,,MUMGCCWCDPRDSS29,,pgshard29,,standby0,,
+plproxy2,100.71.185.12,postgresql,plproxy,,MUMGCCWCDPRDPLPV0,,,,,,
+plproxy3,100.71.185.9,postgresql,plproxy,,MUMGCCWCDPRDPLPV1,,,,,,
+pgsynclog1,100.71.184.252,postgresql,synclog_dbs,,MUMGCCWCDPRDPSLV0,pgsynclog1standby,pgsynclog0,"[""spare"",""standby0""]",,,
+pgsynclog1standby,100.71.184.251,pg_standby,synclog_dbs,elasticsearch,MUMGCCWCDPRDPSLV1,,pgsynclog1,,standby0,,es34
```

Comment:
Changed line is here , added to elasticsearch group and elasticsearch_node_name vars.